src/org/zaproxy/zap/extension/websocket/manualsend/WebSocketPanelSender.java

```diff
@@ -44,9 +46,13 @@
     private static final Logger logger = Logger.getLogger(WebSocketPanelSender.class);
 
 	private Map<Integer, WebSocketProxy> connectedProxies;
+	private Map<Integer, WebSocketProxy> closedProxies;
+	private WebSocketReopenSend webSocketReopenSend;
 
-    public WebSocketPanelSender() {

```

Comment:
Might want to deprecate this, or have it used the new one with a default passed? Just so compatibility isn't broken with anyone else's code.